File: server/hotrod/src/test/scala/org/infinispan/server/hotrod/event/HotRodReplicatedEventsTest.scala
package org.infinispan.server.hotrod.event

import org.infinispan.configuration.cache.CacheMode
import org.testng.annotations.Test

/**
 * @author Galder Zamarreño
 */
@Test(groups = Array("functional"), testName = "server.hotrod.event.HotRodReplicatedEventsTest")
class HotRodReplicatedEventsTest extends AbstractHotRodClusterEventsTest {
   cacheMode = CacheMode.REPL_SYNC
}

File: server/hotrod/src/test/java/org/infinispan/server/hotrod/event/HotRodReplicatedEventsTest.java
package org.infinispan.server.hotrod.event;

import org.infinispan.configuration.cache.CacheMode;
import org.testng.annotations.Test;

/**
 * @author Galder Zamarreño
 */
@Test(groups = "functional", testName = "server.hotrod.event.HotRodReplicatedEventsTest")
public class HotRodReplicatedEventsTest extends AbstractHotRodClusterEventsTest {
   public HotRodReplicatedEventsTest() {
      cacheMode = CacheMode.REPL_SYNC;
   }
}
